deploy/crds/enterprise.splunk.com_clustermasters_crd.yaml

```diff
@@ -2740,11 +2740,11 @@ spec:
                                 format: int64
                                 type: integer
                               appInstallStatus:
-                                description: AppInstallState represents the current
+                                description: AppInstallStatus represents the current
                                   install state of the app
                                 properties:
                                   appInstallState:
```

Comment:
Should this be AppDownloadState instead of AppInstallState. There is a mismatch between the param name and the description. Same applies to other CRDs too

Reply:
huh strange, let me check again.